src/apps/workspaces/crud/roles.py

```diff
@@ -49,8 +51,8 @@ async def get_by_user_id_role_admin(
         self, user_id_: int
     ) -> list[UserAppletAccess]:
         query: Query = select(self.schema_class).filter(
-            self.schema_class.user_id == user_id_
-            and self.schema_class.role == Role("admin")
+            self.schema_class.user_id == user_id_,
+            self.schema_class.role == Role("admin"),
```

Comment:
as I have seen, you can use `Role.ADMIN` also

Reply:
That's exactly right.